rules/imputation.smk

```diff
@@ -7,59 +7,69 @@ rule impute:
     input:
         rules.phase.output,
         refHaps=REF_HAPS
-    output: temp("imputed/chr{chrom}.imputed.dose.vcf.gz")
+    output: temp('imputed/{batch}_chr{chrom}.imputed.dose.vcf.gz')
     log:
-        "logs/impute/minimac4-{chrom}.log"
+        'logs/impute/{batch}_minimac4-{chrom}.log'
     benchmark:
-        "benchmarks/impute/minimac4-{chrom}.txt"
+        'benchmarks/impute/{batch}_minimac4-{chrom}.txt'
     shell:
-        """
+        '''
```

Comment:
I can remember, you said that single quotes (`'`) do not work here. Is it right?

Reply:
`'` does not work inside a shell script, outside of it works fine